nautobot/core/templates/generic/object_bulk_import.html

```diff
@@ -33,84 +34,24 @@ <h1>{% block title %}{{ obj_type|bettertitle }} Bulk Import{% endblock %}</h1>
                         </div>
                     </form>
                     <div class="clearfix"></div>
-                    <p></p>
-                    {% if fields %}
-                        <div class="panel panel-default">
-                            <div class="panel-heading">
-                                <strong>CSV Field Options</strong>
+                    {% include 'generic/inc/bulk_import_field_options.html' %}
```

Comment:
Is there a reason for repeating the `bulk_import_field_options` twice (once within each `tab-pane`) rather than just having it once after the end of the `tab-content` div so that it's always visible?

Reply:
No special reason, I thought it needed to be in the tab  panel. Will change it.